crates/re_renderer/src/queuable_draw_data.rs

```diff
@@ -9,6 +9,9 @@ pub enum QueueableDrawDataError {
     #[error("Failed to retrieve renderer of type {0}")]
     FailedToRetrieveRenderer(&'static str),
 
+    #[error(transparent)]
+    DrawRenderer(#[from] DrawError),
```

Comment:
What's the difference to `DrawError` below, shouldn't that be fully replaced now?

```suggestion
    DrawError(#[from] DrawError),
```